libcaf_core/biicode.conf

```diff
@@ -0,0 +1,42 @@
+# Biicode configuration file
```

Comment:
The biicode project folder for this file is ``libcaf_core`` so bii code expects all reference files needed to build this module within that folder.